index.html

```diff
@@ -0,0 +1,201 @@
+                <li class="lists">Ruby</li>
+                <li class="lists">Html</li>
+                <li class="lists">Css</li>
+              </ul>
+            </div>
+            <div class="frameworks">
+              <h3 class="list-title">Frameworks</h3>
+              <ul>
+                <li class="lists">Bootstrap</li>
+                <li class="lists">Ruby on Rails</li>
+                <li class="lists">RSPec</li>
+                <li class="lists">CapyBara</li>
+                <li class="lists">Selenium</li>
+              </ul>
+            </div>
+            <div class="skills">
+              <h3 class="list-title">Skills</h3>
+              <ul>
+                <li class="lists">Codekit</li>
+                <li class="lists">Github</li>
+                <li class="lists">Codepen</li>
+                <li class="lists">Gitlab</li>
+                <li class="lists">Terminal</li>
+              </ul>
+            </div>
+          </div>
+          <hr class="bottom-divider" />
+          <ul class="social-apps">
+            <li>
+              <a href=""><i class="fa-brands fa-github"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa-brands fa-linkedin-in"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa-brands fa-angellist"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href=""><i class="fa-solid fa-m"></i></a>
+            </li>
+          </ul>
+        </section>
+         <!-- The second section. Main body -->
+
+    <section class="main-body">
+        <div class="work">
+          <i class="fa-solid fa-arrow-down"></i>
+          <h3>Work</h3>
+        </div>
+        <h2 class="recent-work">MY RECENT WORK</h2>
+        <hr />
+        <div class="background-image">
+            <div class="projects-container">
+              <div class="projects">
+                <div class="img-placeholder">
+                  <img src="images/img-placeholder.png" alt="">
+                </div>
+                <div class="project-description">
+                  <h3>Multi-Post Stories</h3>
+                  <p>
+                    A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard dummy text
+                    ever since the 1500s, when an unknown printer took a standard
+                    dummy text.
+                  </p>
+                  <ul class="list-tags">
+                    <li class="border-right">css</li>
+                    <li class="border-right">html</li>
+                    <li class="border-right">bootstrap</li>
+                    <li class="border-left">Ruby</li>
+                  </ul>
+                  <button type="button">See Project</button>
+                </div>
```

Comment:
- [ ] The button ("See project") should follow the button styling: Active, Hover, and Pressed (see design). Please fix the other "see project" buttons also.

![b1](https://user-images.githubusercontent.com/70483835/164940193-8e590d7e-6c40-4af6-aac6-f55ef56d9584.png)